wallet-new/test/unit/UTxO/Verify.hs

```diff
@@ -9,32 +9,32 @@ module UTxO.Verify (
   , verifyBlocksPrefix
   ) where
 
-import Universum
-import Control.Lens ((%=), _Wrapped)
-import Control.Monad.Except
-import Data.Default
-import System.Wlog
+import           Control.Lens ((%=), _Wrapped)
+import           Control.Monad.Except
+import           Data.Default
 import qualified Data.HashMap.Strict as HM
-import qualified Data.HashSet        as HS
-import qualified Data.List.NonEmpty  as NE
-import qualified Data.Map.Strict     as Map
+import qualified Data.HashSet as HS
+import qualified Data.List.NonEmpty as NE
+import qualified Data.Map.Strict as Map
 import qualified Ether
-
-import Pos.Block.Error
-import Pos.Block.Logic hiding (verifyBlocksPrefix)
-import Pos.Block.Pure (verifyBlocks)
-import Pos.Block.Slog hiding (slogVerifyBlocks)
-import Pos.Block.Types
-import Pos.Core
-import Pos.DB.Class (MonadGState(..))
-import Pos.Delegation (DlgUndo(..))
-import Pos.Txp hiding (tgsVerifyBlocks)
-import Pos.Update.Poll
-import Pos.Util.Chrono
-import Pos.Util (ether, neZipWith4)
-import Pos.Util.Lens
-import Serokell.Util.Verify
+import           System.Wlog
+import           Universum
```

Comment:
Please move Universum to the top.

Reply:
Nope, this is a wallet-new playground, we do not interact. I'm sorry that they do not respect `.stylish-haskell` in the project root.

Reply:
You do interact with it in this PR. Since you applied `stylish-haskell` to the imports, you could fix Universum position as well.

Reply:
Feel free to do these kind of `stylish-haskell` formatting in `wallet-new` freely. The reason why the project is still not 100% compliant is that we have several people (new and/or external) needed a bit of adjustment time to deal with the style guides, but the end goal is of course being on track with the rest of the codebase 😉

Reply:
@adinapoli-iohk I'd like to, but the point is that it'll just add conflicts for you guys. I suggest one of people who work on `wallet-new` to do it when they are ready to and set up `stylish-haskell` editor integration at the same time.